### ISSUE TITLE
uninitialized constant Cri::Error

### ISSUE THREAD

**Issue by ostavnaas** [puppetlabs/r10k#853]
When running `r10k puppetfile check`, cri library cast an exception
Version installed cri-2.13.0
Version working cri-2.12.0
r10k version installed 3.0.0
```
Traceback (most recent call last):
        13: from /usr/local/bin/r10k:23:in `<main>'
        12: from /usr/local/bin/r10k:23:in `load'
        11: from /var/lib/gems/2.5.0/gems/r10k-3.0.0/bin/r10k:3:in `<top (required)>'
        10: from /usr/lib/ruby/2.5.0/rubygems/core_ext/kernel_require.rb:59:in `require'
         9: from /usr/lib/ruby/2.5.0/rubygems/core_ext/kernel_require.rb:59:in `require'
         8: from /var/lib/gems/2.5.0/gems/r10k-3.0.0/lib/r10k/cli.rb:3:in `<top (required)>'
         7: from /usr/lib/ruby/2.5.0/rubygems/core_ext/kernel_require.rb:59:in `require'
         6: from /usr/lib/ruby/2.5.0/rubygems/core_ext/kernel_require.rb:59:in `require'
         5: from /var/lib/gems/2.5.0/gems/r10k-3.0.0/lib/r10k/cli/ext/logging.rb:1:in `<top (required)>'
         4: from /usr/lib/ruby/2.5.0/rubygems/core_ext/kernel_require.rb:59:in `require'
         3: from /usr/lib/ruby/2.5.0/rubygems/core_ext/kernel_require.rb:59:in `require'
         2: from /var/lib/gems/2.5.0/gems/cri-2.13.0/lib/cri/command_dsl.rb:3:in `<top (required)>'
         1: from /var/lib/gems/2.5.0/gems/cri-2.13.0/lib/cri/command_dsl.rb:6:in `<module:Cri>'
/var/lib/gems/2.5.0/gems/cri-2.13.0/lib/cri/command_dsl.rb:9:in `<class:CommandDSL>': uninitialized constant Cri::Error (NameError)
Did you mean?  IOError
               Errno
```
I am not sure if this is a r10k issue or cri issue.

I also made an issue in the cri repo
https://github.com/ddfreyne/cri/issues/84

**Comment by denisdefreyne** [comment]
I am getting the same issue here.

Changing (in r10k)

```ruby
# lib/r10k/cli/ext/logging.rb
require 'cri/command_dsl'
```

to

```ruby
# lib/r10k/cli/ext/logging.rb
require 'cri'
```

fixes the issue for me. (Cri is meant to be loaded with `require 'cri'`, and requiring individual files isn’t supported.)

**Comment by denisdefreyne** [comment]
Fix: #854

**Comment by beezly** [comment]
2.6.4 doesn't exhibit this problem for those wanting a temporary workaround without patching.
